src/app/app.component.html

```diff
@@ -21,11 +21,29 @@
                 </li>
             </ul>
         </div>
+
+        <!-- nav bar links after login -->
+        <div class="collapse navbar-collapse justify-content-end" id="navbarNav" *ngIf="router.url == '/dashboard' || router.url == '/group'">
```

Comment:
Same comment as above , create a method in component using angular router service